README.md

```diff
@@ -1,171 +1,192 @@
 # Tuples
 
-<img src="http://i.imgur.com/gMMSTqF.jpg?1" alt="Drawing" style="width: 200px;"/>  
-
+![Drawing](http://i.imgur.com/gMMSTqF.jpg?1)
```

Comment:
Thanks for fixing that for me. I don't know why I originally ran with the above for a while. I had fixed that in later lessons.